src/ContentHub.php

```diff
@@ -583,4 +583,70 @@ protected function getResponseJson(RequestInterface $request)
         $body = (string) $response->getBody();
         return json_decode($body, TRUE);
     }
+
+    /**
+     * Fetch snapshots.
+     *
+     * @return mixed
+     *   Response.
+     *
+     * @throws \Exception
+     */
+    public function getSnapshots()
+    {
+      $endpoint = "$this->api_version/snapshots";
```

Comment:
The only available version for this API call is `/v2`. Then all calls should use that instead of `$api_version`.